projects/addon-editor/components/editor-socket/editor-socket.component.html

```diff
@@ -0,0 +1,24 @@
+<ng-container *ngIf="enablePreview">
+    <ng-template
+        #previewImages
+        let-preview
+    >
+        <tui-preview [rotatable]="true">
+            <img
+                *ngIf="image"
+                loading="lazy"
+                alt="preview-image"
```

Comment:
This should be human readable plain text. I think we can keep it empty `alt=""` for now since this will most likely be used by people with sight and I don't think we want to introduce i18n here.

Reply:
fixed